api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -85,7 +85,7 @@
         return allOfUsCluster;
       };
 
-  private final NotebooksService notebooksService;
+  private final LeonoardoNotebooksClient leonoardoNotebooksClient;
```

Comment:
so the reason I actually made the change is because I wanted to use the name of `NotebooksService` to contain our business logic around notebooks. It seemed like the previous version of `NotebooksService` was mainly acting as a layer between us and leonardo so I renamed it as such.